source/lac/petsc_parallel_sparse_matrix.cc

```diff
@@ -707,6 +707,22 @@ namespace PETScWrappers
       return indices;
     }
 
+    void
+    SparseMatrix::mmult (SparseMatrix       &C,
+                         const SparseMatrix &B,
+                         const MPI::Vector  &V) const
+    {
+      MatrixBase::mmult (C, B, V);
```

Comment:
can you add (here and in all of the other functions you implement) a comment of the form
```
  // simply forward to the protected member function of the base class that
  // takes abstract matrix and vector arguments (to which the compiler automatically
  // casts the arguments)
```

Reply:
Thanks, just added this comment to `SparseMatrix::mmult`, `SparseMatrix::Tmmult`, `MPI::SparseMatrix::mmult` and `MPI::SparseMatrix::Tmmult` and rebased on the latest master.